index.html

```diff
@@ -8,50 +8,266 @@
-                I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
-            </p>
-            <p class="contact">LET’S CONNECT</p>
+            <div>
+                <h1 class="main-header">I’m Ibrahim <br/> Glad to see you!</h1>
+                <p class="main-text">
+                    I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+            </div>
+            <div class="main-items">
+                <p class="contact">LET’S CONNECT</p>
+                <ul class="icons-list list-style ">
+                    <li class="icons">
+                        <a href="https://twitter.com/ibr_ahmat" target="”_blank”" rel="noreferrer noopener"><img src="./assets/twitter.svg" alt="Twitter icon" /></a>
+                    </li>
+                    <li class="icons">
+                        <a href="https://www.linkedin.com/in/ibrahim-ahmat-b5513b1a6/" target="”_blank”" rel="noreferrer noopener"><img src="./assets/linkedin.svg" alt="Linked-In icon" /></a>
+                    </li>
+                    <li class="icons">
+                        <a href="#" rel="noreferrer"><img src="./assets/medium.svg" alt="Medium icon" /></a>
+                    </li>
+                    <li class="icons">
+                        <a href="https://github.com/ibr5500" target="”_blank”" rel="noreferrer noopener"><img src="./assets/github.svg" alt="Github icon" /></a>
+                    </li>
+                    <li class="icons">
+                        <a href="# " rel="noreferrer"><img src="./assets/angellist.svg" alt="Angellist icon" /></a>
+                    </li>
+                </ul>
+            </div>
         </section>
-        <ul class="icons list-style ">
-            <li>
-                <a href="https://twitter.com/ibr_ahmat" target="_blank" rel="noreferrer noopener"><img src="./assets/twitter.svg" alt="Twitter icon" /></a>
-            </li>
-            <li>
-                <a href="https://www.linkedin.com/in/ibrahim-ahmat-b5513b1a6/" target="_blank" rel="noreferrer noopener"><img src="./assets/linkedin.svg" alt="Linked-In icon" /></a>
-            </li>
-            <li>
-                <a href="#" rel="noreferrer"><img src="./assets/medium.svg" alt="Medium icon" /></a>
-            </li>
-            <li>
-                <a href="https://github.com/ibr5500" target="_blank" rel="noreferrer noopener"><img src="./assets/github.svg" alt="Github icon" /></a>
-            </li>
-            <li>
-                <a href="# " rel="noreferrer"><img src="./assets/angellist.svg" alt="Angellist icon" /></a>
-            </li>
-        </ul>
     </main>
+    <section class="card">
```

Comment:
- [ ] It is required that the main container of the projects section has a `display: grid;` property. Kindly add the `display: grid;` property to the main container of the section.